NFIQ2/NFIQ2Algorithm/include/nfiq2_algorithm.hpp

```diff
@@ -48,6 +52,10 @@ class Algorithm {
 	 * The md5 checksum of the provided file.
 	 */
 	Algorithm(const std::string &fileName, const std::string &fileHash);
+#ifdef __ANDROID__
+	Algorithm(AAssetManager *assets, const std::string &fileName,
```

Comment:
Can this be const?

```suggestion
	Algorithm(const AAssetManager *assets, const std::string &fileName,
```

Reply:
When we use the const pointer we would need to do an internal const cast, due the NDK function do not use a const pointer.

Reply:
Okay, no `const` then. Casting would defeat the purpose 😀